src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -500,6 +507,17 @@ bool RendererOpenGL::Init() {
 
     return true;
 }
+void RendererOpenGL::FrameLimiter(float average_frame_time,int framelimit)
+{
+    //calculate  difference between average frame time and frame time for the framelimit, and sleep for the difference in time.
+    float frame_time = 1000 / framelimit;
+    if (average_frame_time < frame_time)
+    {
```

Comment:
FYI, the travis logs will tell you all the formatting changes that need to be made https://travis-ci.org/emmauss/citra/jobs/175323259 (in case you don't want to set up clang-format locally)